tests/openssh.rs

```diff
@@ -292,6 +292,31 @@ async fn stdout() {
     }
 }
 
+#[tokio::test]
+#[cfg_attr(not(ci), ignore)]
+async fn over_session() {
```

Comment:
I'd like to see this also test some cases that require escaping.

Reply:
I'm not quite sure what a good test case should look like here and I'd greatly appreciate a hint. :thinking:

Reply:
Ah, so, I was thinking things like having an argument that contains, say, `'`, ` `, and `\`.